client/blocks/installments-plan-picker/index.jsx

```diff
@@ -0,0 +1,93 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+
+export class InstallmentsPlanPicker extends React.Component {
+	static propTypes = {
+		initialValue: PropTypes.number,
+		plans: PropTypes.array.isRequired,
+		onChange: PropTypes.func,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		onChange: () => null,
+	};
+
+	state = {
+		installmentsInputValue: this.props.initialValue,
+	};
+
+	formatPlans() {
+		return this.props.plans.map( plan => {
+			plan.label = plan.value + ' x ' + plan.payment;
+			return plan;
+		} );
+	}
+
+	handleInstallmentsInput = event => {
+		const installments = parseInt( event.target.value );
+
+		this.setState( {
+			userChangedInstallments: true,
+			installmentsInputValue: installments,
+		} );
+
+		this.props.onChange( installments );
+
+		this.setState( {
+			userChangedInstallments: false,
+		} );
+	};
+
+	render() {
+		const installmentPlans = this.formatPlans();
+		return (
+			<div className="installments-plan-picker">
+				<div className="installments-plan-picker__header">
+					{ this.props.translate( 'Choose an installments plan', {
```

Comment:
I think the common wording is either **Choose an installment plan** or **Pay by (or in) installments**. 

Does the payment provider offer any additional information that we could display to the user in this section to explain what's going to happen, or what these means for their bank account? :)

Reply:
Also, maybe a design person can step in here, would it be too much to specify the payment amount in the button?

<img width="463" alt="screen shot 2018-05-18 at 1 45 29 pm" src="https://user-images.githubusercontent.com/6458278/40215101-d9fabf44-5aa1-11e8-8572-72c3ef44ce22.png">